Maple2.Server.Core/proto/channel/channel.proto

```diff
@@ -105,3 +107,52 @@ message PartyRequest {
 message PartyResponse {
   int32 error = 1;
 }
+
+message GroupChatRequest {
```

Comment:
This seems duplicated, you can put in `common.proto` and then import it.

See: `ChatRequest`/`BuddyRequest`

Reply:
Only some overlap. (ex. Create in world only or AddMember in channel only). Should I move the duplicated ones to common?

Reply:
Doesn't moving it to common make it easier to forward the request (since you don't need to translate between two protos) if so I think it makes sense even if it's only a subset.